src/serac/physics/utilities/functional/tests/functional_boundary_test.cpp

```diff
@@ -29,6 +29,65 @@ constexpr bool verbose     = true;
 std::unique_ptr<mfem::ParMesh> mesh2D;
 std::unique_ptr<mfem::ParMesh> mesh3D;
 
+double relative_error_frobenius_norm(const mfem::SparseMatrix & A, const mfem::SparseMatrix & B) {
+  if (A.Height() != B.Height()) return false;
+  if (A.Width()  != B.Width())  return false;
+
+  double fnorm_A = 0.0;
+  double fnorm_A_minus_B = 0.0;
+
+  for (int r = 0; r < A.Height(); r++) {
+    auto columns = A.GetRowColumns(r);
+    for (int j = 0; j < A.RowSize(r); j++) {
+      int c = columns[j];
+      if (c < 0) std::cout << "??" << std::endl;
```

Comment:
curly braces and a more helpful message.. is this an failure?

Reply:
Yes, this should be a `SLIC_ERROR`

Reply:
an old debugging comment. Removed!